model/ply.py

```diff
@@ -0,0 +1,20 @@
+from model.point import Point
+from typing import Optional
+
+
+class Line:
+    points: list[Point]
+
+    def __init__(self, points: list[Point]) -> None:
+        self.points = points
+
+
+class Ply:
```

Comment:
i would prefer that you don't call this `Ply` given Ply has a very specific standard and this isn't following that. Shouldn't this sit as an internal class to Serializer? We should never explicitly access or instantiate this as it's a helper class to Serializer